src/main/java/org/vietabroader/controller/WebcamController.java

```diff
@@ -19,12 +20,16 @@
 import com.google.zxing.Result;
 import com.google.zxing.common.HybridBinarizer;
 import org.vietabroader.model.GlobalState;
+import org.vietabroader.model.VASpreadsheet;
+import sun.jvm.hotspot.runtime.Threads;
 
 public class WebcamController implements Controller {
     private static final Logger logger = LoggerFactory.getLogger(AuthenticationController.class);
 
     private final int READER_SLEEP_TIME_MS = 1000;
 
+    private List<Object> participantList;
```

Comment:
If this is not used anywhere else other than in `process()`, declare it there instead of class scope.